CMakeLists.txt

```diff
@@ -182,6 +182,7 @@ if(BUILD_CLIENT)
       pkg_check_modules(CLOUDPROVIDERS cloudproviders IMPORTED_TARGET)
 
       if(CLOUDPROVIDERS_FOUND)
+        pkg_check_modules(DBUS-1 REQUIRED dbus-1 IMPORTED_TARGET)
```

Comment:
sorry for the late review
I do not really get why we check for something we will not use `dbus-1` is never used
we do not use it when linking and we never use it to locate include headers

so my next question is why do you want that check ?

Reply:
No worries!

The presence of `dbus-1` seems to be expected in [shell_integration/libcloudproviders/CMakeLists.txt](https://github.com/nextcloud/desktop/blob/40021e69438846b24e7c801dbca604aefbaf5a79/shell_integration/libcloudproviders/CMakeLists.txt#L22-L28). If you have `cloudproviders` but don't have `dbus-1`, then you get a build error that doesn't say what exactly is missing and you have to dig through the build files in order to figure it out. See also #4153.

Reply:
sorry missed this access to dbus-1 package
so thanks a lot for the fix !!!